client/me/purchases/purchases-list-in-dataviews/purchases-data-view.tsx

```diff
@@ -23,7 +23,7 @@ export function PurchasesDataViews( props: {
 } ) {
 	const { purchases } = props;
 	const onChangeView = () => {
-		alert( 'You clicked something!!' );
+		return;
```

Comment:
As onChangeView does not do anything, I would have personally added an empty method directly in the component as this seems to be the approach [in other places](alypso/blob/7cd2ea9673cad781fea277a42a0a4e455dea51ba/apps/notifications/src/panel/templates/index.jsx#L526). It helps uncluttering the code a little. But this works as well.

```
<DataViews
   ...
   onChangeView={ (view: View) => {} }
   ...
/>
```